### ISSUE TITLE
Expose the raw data through the interface

### ISSUE THREAD

**Issue by frbl** [roqua/physiqual#47]

**Comment by emerencia** [comment]
How should this be implemented? If we want to return the "raw" raw data, then we can only export one data source (e.g., heart rate or steps) at a time because there is no clear way to interleave the data otherwise. We could add a method to oauth_session_controller called raw_data that looks for a params[:data_source] and and a params[:service_provider] and returns whatever we receive from the Google/Fitbit service providers, but there are more caveats.

We can't simply create a RawExporter, since the base Exporter class has all of the aggregate and bucket logic, which we don't want. But if we want the returned measurements to be the exact same set that our aggregate measurements are based on, we need to do some date/time interpretation to e.g., remove measurements that are from x hours before the first measurement, or after the last measurement, instead of simply including all the measurements on the first and last day.

We could just skip the data aggregators, define a from and a to as:
```ruby
        from = first_measurement - Physiqual.hours_before_first_measurement.hours
        to   = first_measurement + (number_of_days - 1).days +
               ((Physiqual.measurements_per_day - 1) * Physiqual.interval).hours
```
and then simply call the Google or Fitbit DataService directly. However, at least the Fitbit service returns all measurements on both the beginning and end day. Removing measurements from before x hours before the first measurement and after the last measurement is done in the SummarizedDataService which looks at the timestamps which it takes from the BucketGenerator's buckets. And since this can also change depending on whether we use an EquidistantBucketGenerator or a daily schedule, I don't think this logic should be moved.

In short, the period of measurements to include prior to the first measurement depends on the buckets and therefore I think we should just return all data on the beginning and end day for a raw data request, the requester can then remove or include the measurements themselves.

**Comment by emerencia** [comment]
Started work in #50.

**Comment by emerencia** [comment]
Fixed.
